tests/SpecTests/ClientSideEncryptionSpecTest.php

```diff
@@ -2001,28 +1995,4 @@ private function prepareCorpusData(string $fieldName, stdClass $data, ClientEncr
 
         return $data->allowed ? $returnData : $data;
     }
-
```

Comment:
@eramongodb, @kevinAlbs: On a related note, our legacy CSFLE spec test runner also requires at least one of `crypt_shared` and `mongocryptd` to be available, but I don't see that explicitly mentioned in [Use as integration tests](https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests#use-as-integration-tests). I understand why they _would_ be required since various tests due cover automatic encryption.

Am I just missing the text, or is it only implied?

Reply:
I think it is implied, but the test README may benefit from edits:

> Do the following before running spec tests:
>
> - If available for the platform under test, obtain a crypt_shared binary and place it in a location accessible to the tests. Refer to: Using crypt_shared
> - Start the mongocryptd process.

This may benefit from edits. I expect mongocryptd does not need to be started, only available on the PATH for drivers to spawn. If testing with crypt_shared, I expect mongocryptd does not need to be available.